netboot/main.go

```diff
@@ -108,6 +110,16 @@ func main() {
 		}
 		for _, d := range dhcp {
 			if err := boot(iface.Name, d); err != nil {
+				if !*noFixmynetboot {
+					cmd := exec.Command("fixmynetboot", "-shell", iface.Name)
```

Comment:
Does it drops into an emergency shell if one of the interfaces fails a check?

Reply:
Yes, good catch again. I'll drop the `-shell` parameter.
Thing is - I'm preparing a demo where I need the DHCP check plus others that should not be the default, and some demo code is left in. Sorry about that.